.gitignore

```diff
@@ -130,3 +130,7 @@ dmypy.json
 
 # Typings generated by VSCode
 typings/**
+
+# Libraries
+!libbls.so
```

Comment:
since this file is 8MB, can we use some kind of lazy loading? download it from somewhere (not sure where) only at the first usage of the bls signing?
Not sure if possible or 8MB means a lot or not compared to the rest of the module

Reply:
Very good question / issue. These libraries are, indeed, the largest objects of the package.

However, the library installation would normally only take a few seconds. 

erdpy (the CLI tool) implements lazily downloading in several cases - though, I'd vote in favor of `erdpy-wallet` being standalone, at least for the moment - and monitor eventual issues that might arise.